internal/cmd/firewall/apply_to_resource_test.go

```diff
@@ -43,7 +43,7 @@ func TestApplyToServer(t *testing.T) {
 
 	out, errOut, err := fx.Run(cmd, []string{"--type", "server", "--server", "my-server", "test"})
 
-	expOut := "Firewall 123 applied\n"
+	expOut := "Firewall 123 applied to resource\n"
```

Comment:
We might target many resources when using the label selector, but I think this is good enough.